src/Ch2/PositiveNegative.java

```diff
@@ -0,0 +1,26 @@
+//Program to check no is positive or negative
+/**
+ * @author Naresh
+ * Year 2018
+ * Version 1.1
+ */
+
+package Ch2;
+
+
+public class PositiveNegative {
+
+	public static void main(String[] args) {
+		
+		int num1;
+		num1 = -35;
+		
+		if (num1 >= 0)
+		{
+			System.out.println("Number is Positive");
+		}
+		else System.out.println("Number is Negative");

```

Comment:
Proper indentation for else block with curly braces makes a programmable easy to read.